src/utils/parsers/common.ts

```diff
@@ -360,11 +361,18 @@ export abstract class BaseParser {
 
         for (const match of matches) {
             const usage = this.extractUsageInformation(filteredFile, match)
+
+            const bufferedContent = file.lines
+                .filter((line) => usage.start - buffer <= line.ln && usage.end + buffer >= line.ln)
+                .map(line => line.content)
+                .join('\n')
+            
             result.push({
                 name: match.name,
                 line: usage.start,
                 fileName: file.name,
-                content: usage.content
+                content: usage.content,
```

Comment:
do we need both types of content? I think just the buffered content may be sufficient